### ISSUE TITLE
Input still has error styling after changing error conditions

### ISSUE THREAD

**Issue by DefinitelyNotAWalrus** [Stegosource/vuetensils#151]
On our site, we are using the Vuetensils input in several places. On one page, we have two inputs, one for a number of days and one for the user to enter a date. The maximum number of days is dependent on the date entered. We have been using the errors prop to handle any errors with this.
We recently noticed that when you enter a number of days that is out of the maximum range, it will expectedly display an error message. However, if you then change the date to later (to accommodate the new range of days), the error will now acknowledge the new maximum in text, but still have the .vts-input-error styling as if there is still an error. I would expect that the red border would no longer show.
Basically, if a value is outside the maximum, an error shows up as it should. If the conditions for throwing the error change, the error styling ought to disappear, but it doesn’t. We were looking for a solution to this.

**Comment by AustinGil** [comment]
Thanks for reporting, and thanks for using the library. I hope you like it :)

Do you mind sharing a URL to the project, or even better, a reproduction in Codesandbox or Stackblitz?

**Comment by DefinitelyNotAWalrus** [comment]
Reproduced it here. Hopefully I explained things clearly enough in there.

[https://codesandbox.io/s/compassionate-architecture-h2xtqd?file=/src/components/HelloWorld.vue:711-762](url)

**Comment by AustinGil** [comment]
OK, this is very helpful. Yes, it's clear. I see the issue.

I'll take a look and see what I can do. My initial suspicion is that Vue is responding to its own synthetic event system to update the UI, but because Vuetensils uses native events, the change may not be triggered on the target input, which means the re-validation doesn't run.

I might be able to get around this by explicitly redefining the HTML validation attributes as props which would trigger a re-render, or maybe I can set up some sort of native attribute watcher.

I'm not sure, but I've got some decent ideas where to start.

Thanks again for using the library and helping me improve it :)

**Comment by AustinGil** [comment]
@DefinitelyNotAWalrus, can you pull the latest version and see if that fixes the issue for you?

**Comment by AustinGil** [comment]
Closing this issue unless I hear back.

**Comment by DefinitelyNotAWalrus** [comment]
Hello, sorry for the late response. My email notifications were not bumping up to the top, so I wasn't aware that you had been responding.
I just updated to the latest version and am still seeing the same issue.

**Comment by AustinGil** [comment]
Hmm. That's weird. I did have it working locally, but you're right that it's still not working in that demo. I'll need a bit more time on this.

**Comment by AustinGil** [comment]
Ok, this one was a two parter. I got the attr watcher in place last time. But missed the part where I had to wait for the next event loop cycle for the DOM to be updated before validating. Give it a go and let me know @DefinitelyNotAWalrus

**Comment by DefinitelyNotAWalrus** [comment]
That did seem to work for me in Code Sandbox. Thank you!
